server/app/views/components/AlertFragment.html

```diff
@@ -0,0 +1,36 @@
+<div
+  th:fragment="alert(alertSettings, headingLevel)"
+  th:class="${'usa-alert usa-alert--' + alertSettings.alertType.name().toLowerCase()}"
+  aria-live="polite"
+  role="alert"
+>
+  <div class="usa-alert__body">
+    <h2
+      class="usa-alert__heading"
+      th:if="${alertSettings.title.isPresent() && headingLevel == 'H2'}"
```

Comment:
**question:** It doesn't seem ideal to me that we would have so many different heading levels for our alerts.  Are we using all of these now?  If we limit the header levels here, it would encourage people to be consistent.  Would you mind checking with Sijia on what the ideal header levels should be?

Reply:
I talked with Sijia before about having to move it at one point because of one place was failing accessibility tests due to the ordering of H1-6 tags or because some pages had a gap. Which were also different between current and northstar >_< 

Unless we only ever use the non-slim ones in one place with no other headings deviating we'll always need the ability to override.

I don't disagree with you, we should make it easy to be consistent. I don't want to block this PR though, we should get with Don and Sijia to make sure NS is doing H1-6 correctly as a whole.

Reply:
That sounds good.  I think Sijia already has a work item to review our headings and make sure they're consistent, so, hopefully, they'll review the alert headings as part of that.